Wikipedia/Code/SearchViewController.swift

```diff
@@ -65,7 +65,20 @@ class SearchViewController: ArticleCollectionViewController, UISearchBarDelegate
     
     @objc var areRecentSearchesEnabled: Bool = true
     @objc var shouldBecomeFirstResponder: Bool = false
-    
+
+    var displayType: NavigationBarDisplayType = .largeTitle
+    var shouldSetSearchVisible: Bool = true
+    var shouldSetTitleViewWhenRecentSearchesAreDisabled: Bool = true
+
+    var shouldShowCancelButton: Bool = true
+    var delegatesSelection: Bool = false {
+        didSet {
+            resultsViewController.delegatesSelection = delegatesSelection
+        }
+    }
+
+    var showLanguageBar: Bool?
```

Comment:
I ran into an issue with multiple languages where I was editing an article in a different language but I could only add links from my default language. It makes sense to have the language bar hidden but there should probably be a way to set the site to be searched. This way, `EditLinkViewController` and `InsertLinkViewController` could pass in the site of the article being edited so that the search is performed on that site.

Reply:
Good point, thanks!

Reply:
Something like https://github.com/wikimedia/wikipedia-ios/pull/3076/files/78d95ac0b070a1ce03afb5e1f08fa0e96adf8af1..4bf211294acc9dd4aa01b3d19a62df800e3b2ec0?

Reply:
yup! looks good